src/Calendars/Islamic/arafat-holidays.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+return [
+
+    'jo' => [
```

Comment:
Can you use class consts for this? Have a look at Egypt or Bahrain

Reply:
@Nielsvanpach 
Why I used this approach:

1. It's hard to read around 200 lines of code to read the elements.
2. in some countries the holidays are shared or in the same region.
3. at the end the Islamic directory is loadable.

Reply:
2 and 3 seem valid points.

Although, using `jo` as a key doesn't feel like the way to go for being shareable across countries?

Also, if you have a look at Turkey, I prefer that format. What the benefit of using the extra `date` key? 

Also, for multiday holidays it's possible to have it twice in a single year:

```php
1999 => '01-19',
2000 => [
    '01-08',
     '12-27',
],
```

Reply:
`jo` it may change,

`date` is more readable,

The third note is valid.